### DISCUSSION TITLE
DataTables support to MongoDB

### DISCUSSION THREAD

**Discussion by lalithhakari** [yajra/laravel-datatables#2881] [General]
Hi, I'm using Laravel 9 + [jenssegers/laravel-mongodb](https://github.com/jenssegers/laravel-mongodb) +  yajra/laravel-datatables
Im not able to integrate Laravel-datatables with mongodb. Any help/suggestion is needed.

**Comment by redati** [comment]
yes https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/upgrade/
